lib/omniauth/strategies/openid_connect.rb

```diff
@@ -262,6 +264,11 @@ def access_token
         @access_token = client.access_token!(token_request_params)
         verify_id_token!(@access_token.id_token) if configured_response_type == 'code'
 
+        if @access_token.expires_in
+          @access_token_expires_at = Time.now.to_i + @access_token.expires_in
```

Comment:
Can we use the `AccessToken#expires_at` method with an `expires_latency` option?

Reply:
I guess the tl;dr answer to your question is, "yes... should we?"  🙂 

I don't think `Rack::OAuth2::AccessToken` _has_ an `#expires_at` method... https://github.com/nov/rack-oauth2/blob/master/lib/rack/oauth2/access_token.rb, or am I looking in the wrong place?  I could submit a PR for that upstream.

This is the question that I was referring to in the PR description:
> I'm not sure if the best place to fix it is here or upstream, in 
> Rack::OAuth2::AccessToken.  On the one hand, the oauth2 gem handles it in
> OAuth2::AccessToken.  On the other hand, the OmniAuth strategy is the only place we can
> ensure minimal latency between the server response and expires_at computation.  I chose
> here. 🙂

That was the justification I gave on Dec 6th.  But, in retrospect, I may have also been motivated to do it this way because it meant one PR in one repo rather than two in two repos. 😉

It makes intuitive sense to me that `expires_latency` should be an optional parameter to `#expires_at`, like you suggest.  I think one benefit of applying `expires_latency` during construction might be that it allows code that _uses_ access_token to remain naive, and thus simplifies application of a uniform policy.  🤷🏻  I don't feel strongly about it one way or the other, and I'll gladly apply `expires_latency` in the initializer (like the `oauth2` gem), as an argument to the `#expires_at` method, or both.

What do you think?  Should I make a PR to https://github.com/nov/rack-oauth2 first, and then update this PR after a new version is released there?